velox/docs/configs.rst

```diff
@@ -524,6 +524,19 @@ Each query can override the config by setting corresponding query session proper
      - bool
      - false
      - Utilize the configuration of the environment variables http_proxy, https_proxy, and no_proxy for use with the S3 API.
+   * - hive.s3.connect-timeout
+     - string
+     - 1s
```

Comment:
Looking at other optional configs, the convention is to leave this empty.
Same below.